frontend/app/components/Spots/SpotPlayer/components/SpotVideoContainer.tsx

```diff
@@ -170,24 +173,43 @@ function SpotVideoContainer({
     }
   }, [spotPlayerStore.playbackRate]);
 
-  const warnText = isProcessing ? 'You’re viewing the entire recording. The trimmed Spot is on its way.' : 'Your trimmed Spot is ready! Please reload the page.'
+  const reloadPage = () => { window.location.reload();  };
+
+  // const warnText = isProcessing ? 'You’re viewing the entire recording. The trimmed Spot is on its way.' : 'Your trimmed Spot is ready!'
   return (
     <>
-      {isProcessing || prevIsProcessing
-       ? <div
-         className="px-3 py-1 border border-gray-lighter drop-shadow-md rounded bg-active-blue flex items-center justify-between"
-         style={{
-           zIndex: 999,
-           position: 'absolute',
-           left: '50%',
-           top: '-24px',
-           transform: 'translate(-50%, 0)',
-           fontWeight: 500
-         }}
-       >
-         {warnText}
-       </div>
-       : null}
+      {isProcessing ? (
+        <div className="absolute z-20 left-2/4 -top-6" style={{ transform: 'translate(-50%, 0)' }}>
+          <Alert
+            className='trimIsProcessing rounded-lg shadow-sm border-indigo-500 bg-indigo-50'
+            message="You’re viewing the entire recording. The trimmed Spot is on its way."
+            showIcon
+            type="info"
+            closable
+            icon={<InfoCircleOutlined style={{ color: '#394dfe' }} />}
+          />
+        </div>
+      ) : prevIsProcessing ? (
+        <div className="absolute z-20 left-2/4 -top-6" style={{ transform: 'translate(-50%, 0)' }}>
+          <Alert
+            className='trimIsReady rounded-lg shadow-sm border-0'
+            message="Your trimmed Spot is ready!"
+            showIcon
+            type="success"
+            action={
+              <Button
+                size="small"
+                type="text"
+                icon={<PlayCircleOutlined />}
+                onClick={reloadPage}
+              >
+                Play Now
```

Comment:
"play now" would be applicable if it was not playing at all, maybe "reload" or something like that?